app/index.html

```diff
@@ -20,7 +20,7 @@
     <script type="application/javascript">var baseUrl = "<%= BASE_URL %>"</script>
 </head>
 <body data-color-scheme="auto">
-<noscript><%= require('@/assets/global-elements/noscript.html') %></noscript>
+<noscript><%= require('@/assets/global-elements/noscript.html').default %></noscript>
```

Comment:
Thanks for the `.default` fix! That must be a fairly recent regression since it wasn't necessary in the past 😅

On another note, I assume that your other PR that does the pre-rendering would actually need to remove this `<noscript>` element entirely, right? Otherwise, we end up with something like this where it is included at the top of the page instead of the current experience where it is the full page (without JS enabled).

<img width="1436" alt="Screenshot 2023-12-07 at 2 03 24 PM" src="https://github.com/apple/swift-docc-render/assets/212918/ee68433c-d929-4f58-87f7-370f80fcaf74">

Reply:
Yep! It will remove that `<noscript>` — although maybe there still should be a bar at the top of the page that warns folks they might not get the full experience if JS is off.